blue_scribe.tex

```diff
@@ -12,17 +12,100 @@
+ 
+\subsection{USB UART Communication Test}
+The USB UART communication test was performed by connecting the microcontroller to a PC via a mini USB and disconnecting the laser and both actuators pins. Once connected, the PC sent commands serially to the microcontroller via a PuTTY terminal to ensure the microcontroller was receiving the commands and sending the proper response flag back to the terminal. We observed that all valid function calls were returned with the proper done flag and all invalid inputs were returned with the proper error flag. This test verified the microcontroller can communicate serially with the PC via USB and can send a done or error flag via the UART back to the PC upon instruction completion or failure.
+
+\subsection{Image Instruction Breakdown Test} Nick talk about this one because idk how to word it
+The image instruction breakdown test was performed on the PC by . This test verified our program can accurately turn images into valid function commands. 
+
+\subsection{Manual Instruction Input Test}
+The manual instruction input test was performed by connecting the microcontroller to a PC via a mini USB and connecting the laser and both actuators pins. Once connected, the PC sent commands serially to the microcontroller via a PuTTy terminal to ensure the microcontroller was receiving the commands and implementing the function calls correctly. We sent the following instructions: GO 5000, 5000; HM; GO 1000, 1000; BH 1000, 500; BV 1000, 500; GO 2000, 2000; SQ 1000, 500; and HM. We observed the actuators and the laser working as expected for each function call and also observed the done flag being sent back to the terminal on each successful instruction completion. This test verified the microcontroller can receive instructions manually through a terminal on the PC.
+
+\subsection{Computer Generate Input Test} Nick talk about how the instructions were sent.
+The manual instruction input test was performed by connecting the microcontroller to a PC via a mini USB and connecting the laser and both actuators pins. Once connected, . This test verified the microcontroller can receive instructions automatically from the PC via the ***** program.
+
+\subsection{Individual Function Test} I don't know if we wanna do this one.
+The individual function test was performed by connecting the microcontroller to the laser and actuators and hard coding and running all of the different commands.  
+
 \section*{Conclusion}
 
-% \begin{figure}[H]
-%   \begin{center}
-%     \includegraphics[width=0.6\textwidth]{blocks}
-%     \caption{Top level design for the flow of interrupts and timers.}
-%     \label{fig:blocks}
-%   \end{center}
-% \end{figure}
+
+\section*{Appendices}
```

Comment:
Figures are in-line this time. Instead of the appendix, they go adjacent to the text that is most relevant.